docs/workflows/input_output.md

```diff
@@ -287,74 +274,72 @@ The CMA will attempt to pull the file stored at `Bucket`/`Key` and replace the v
 
 In the workflow configuration (defined under the `task_config` key), each task has its own configuration, and it can use URL template as a value to achieve simplicity or for values only available at execution time. The Cumulus Message Adapter resolves the URL templates (relative to the event configuration key) and then passes message to next task. For example, given a task which has the following configuration:
 
-```yaml
-    Discovery:
-      Parameters:
-        cma:
-          event.$: '$'
-          task_config:
-            provider: '{$.meta.provider}'
-              inlinestr: 'prefix{meta.foo}suffix'
-              array: '{[$.meta.foo]}'
-              object: '{$.meta}'
+```json
+Discovery:
+{
+  "Parameters": {
+    "cma": {
+      "event.$": "$",
+      "task_config": {
+        "provider": "{$.meta.provider}",
+        "inlinestr": "prefix{meta.foo}suffix",
+        "array": "{[$.meta.foo]}",
+        "object": "{$.meta}"
+      }
+    }
+  }
+}
 ```
 
 *and* and incoming message that contains:
 
 ```json
-      "meta": {
-        "foo": "bar",
-        "provider": {
-          "id": "FOO_DAAC",
-          "anykey": "anyvalue"
-        }
-      }
+"meta": {
```

Comment:
This isn't a full JSON object in this snippet. Your call if it should be for consistency

Reply:
Fixed (plus some minor formatting nits): 4a465b818

Reply:
9384f88a7